docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+need a flexible solution which enables them to integrate and connect _any
+system_ and comes with fully-fledged user management and auditing capabilities.
+Lightning relies on the same tried-and-trusted core technology as the OpenFn
+iPaaS and comes with an improved, visual interface for building integrations.
+
+:::note
+
+Any integration built on the OpenFn iPaaS is can be run on Lightning.
+
+:::
+
+## Guiding principles
+
+Lightning is developed in line with the
+[principles for digital development](https://digitalprinciples.org/principles/)
+and under the guidance of it's Open Source Steering Committee which you can read
+about [here](https://openfn.github.io/governance/OSSC.html).
+
+On top of this, Lightning follows 4 key principles which determine how it should
+be developed:
+
+### Integrations break
+
+Anyone that has worked on integration projects in the past is well aware that
+integrations do break. No matter how well designed they are, the fact is that
+**they connect multiple systems that all change over time**: new API versions
+get released, data models change, IDs, codes and mappings change, data standards
+are updated and the processes themselves evolve. This is why Lightning will
+include:
+
+Enhanced testing and debugging:
+
+- Save inputs as edge cases for robust workflow testing
+- View the input and output for each step in a workflow to easily identify where
+  an error occurred
+- Throw custom errors to improve API messages (adaptors)
+- Add custom logic to handle a step failure (fail triggers)
+
+First of class monitoring:
+
+- Get notified on run failures
+- View the status of every run
+- Search workflow runs by input/output data and logs
+- Filter workflow runs by status, workflow name and date
+
+Re-processing functionality:
+
+- Bulk reprocess requests to address any issues or process requests with updated
+  logic
+
+### Collaboration is key
+
+On one hand, the users that understand what processes need automating are (more
+often than not) business analysts, not developers. They’re the experts on what
+needs to happen when and where, and they’re very capable of planning out
+integrations and putting together mapping specifications and bpmn flows.
+
+On the other hand, integrations often require custom logic that cannot be
+simplified through low-code and therefore must be implemented by software
+engineers.
+
+That’s why **the best integrations are built when non-technical users and
+developers collaborate**. Lightning is being developed to bridge the gap between
+non-technical and technical users through:
+
+Intuitive, user-friendly user interface for non-technical users:
+
+- Understand a workflow in a visual, human-readable format (abstract away from
+  code to make workflows understandable to non-technical users)
+- Build credentials through a form (remove the need to read through confusing
+  API documentation)
+- Build requests through a form
```

Comment:
Again reading this as I don't know anything about OpenFn... 
What are `requests`? HTTP requests? 
And "through a form" ... `through a form _interface_` maybe?